### PR TITLE
rtt updates/localization for the Miners' Village

### PR DESCRIPTION
**Pull request by terrapin47**

### CODE REVIEW

File: QUEST_JOBSTEP.tsv [hunk 1]
```diff
@@ -1375,7 +1375,7 @@ QUEST_JOBSTEP_20150317_001374	Talk to the Cryomancer Master
 QUEST_JOBSTEP_20150317_001375	Talk to Cryomancer Mater in Klaipeda.
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: Mater -> Master

File: QUEST_JOBSTEP.tsv [hunk 2]
```diff
@@ -1375,7 +1375,7 @@ QUEST_JOBSTEP_20150317_001374	Talk to the Cryomancer Master
...
 QUEST_JOBSTEP_20150317_001376	{memo X}Chose the path of Cryomancer. Talk to the Cryomancer Master in Klaipeda.
 QUEST_JOBSTEP_20150317_001377	Ask the Miner's Village mayor about the whereabouts of the book
-QUEST_JOBSTEP_20150317_001378	Cryomancer says he's looking for a precious book and he'll help you become a Cryomancer if you find the book for him. Ask the Mayor in Siauliai Miners' Village about the book.
+QUEST_JOBSTEP_20150317_001378	Cryomancer says he's looking for a precious book and he'll help you become a Cryomancer if you find the book for him. Ask the Mayor at the Miners' Village about the book.
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: I assume this is "The Cryomancer Master..."

File: QUEST_JOBSTEP.tsv [hunk 3]
```diff
@@ -1627,10 +1627,10 @@ QUEST_JOBSTEP_20150317_001626	You've succeeded in defeating Sparnas Man. Report
 QUEST_JOBSTEP_20150317_001627	Talk to the Priest Master in Klaipeda.
 QUEST_JOBSTEP_20150317_001628	{memo X}Get the pouch with evil aura from the Vubbes in Crystal Mines 1F
 QUEST_JOBSTEP_20150317_001629	{memo X}The Priest Master ask you to defeat the Vubbes occupying Crystal Mine 1F as a proof of your faith
-QUEST_JOBSTEP_20150317_001630	Imbued Magi Pouch
-QUEST_JOBSTEP_20150317_001631	should be obtained.
-QUEST_JOBSTEP_20150317_001632	Hand it over to the Priest Master
-QUEST_JOBSTEP_20150317_001633	Gathered the pouches with evil aura. Take it to the Priest Master.
+QUEST_JOBSTEP_20150317_001630	Imbued Magic Pouch
+QUEST_JOBSTEP_20150317_001631	 from the Vubbes.
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv:1631]: Magic -> Evil Energy

File: QUEST_JOBSTEP.tsv [hunk 4]
```diff
@@ -2066,12 +2066,12 @@ QUEST_JOBSTEP_20150317_002065	Defeat the Mushcaria in Siauliai West Woods
 QUEST_JOBSTEP_20150317_002066	Prove your strength to the Cryomancer Master by defeating the Mushcaria in Siauliai West Woods.
 QUEST_JOBSTEP_20150317_002067	Defeated Mushcaria in Siauliai West Woods. Go back and report to the Cryomancer Master.
 QUEST_JOBSTEP_20150317_002068	Talk to the Psychokino Master in the town of Shaururei mine.
-QUEST_JOBSTEP_20150317_002069	Defeat the Vubbe Figher in Siauliai Miners' Village
+QUEST_JOBSTEP_20150317_002069	Defeat the Vubbe Figher at the Miners' Village
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: Figher -> Fighter

File: QUEST_JOBSTEP.tsv [hunk 5]
```diff
@@ -2841,8 +2841,8 @@ QUEST_JOBSTEP_20150714_002840	Talk to the Thaumaturge Master in Gateway to Great
 QUEST_JOBSTEP_20150714_002841	Obtain resins of Varvs, tail fragments of Zinutekas and activating nuclues of Boowookes from the 1st floor of Mausoleum.
 QUEST_JOBSTEP_20150714_002842	Talk to the Scout Master in Gateway to Great King.
 QUEST_JOBSTEP_20150714_002843	Talk to the Cataphract Master in Gateway to Great King.
-QUEST_JOBSTEP_20150714_002844	Obtain the demonic pokets from the Vubbes in Siaulai Mine Village.
-QUEST_JOBSTEP_20150714_002845	The Priest Master asked you to defeat Vubbes around Siauliai Miners' Village to prove your sincerety.
+QUEST_JOBSTEP_20150714_002844	Obtain the Imbued Magic Pouches from the Vubbes at the Miner's Village.
+QUEST_JOBSTEP_20150714_002845	The Priest Master asked you to defeat Vubbes around the Miners' Village to prove your sincerity. You should gather
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv:2845]: Magic -> Evil Energy

File: ETC.tsv [hunk 6]
```diff
@@ -7497,7 +7497,7 @@ ETC_20150317_007497	Obtain Purified Essence
 ETC_20150317_007498	Attach the Holy Bomb
 ETC_20150317_007499	Collect the mysterious stone
 ETC_20150317_007500	{memo X}Information for the barrier (3)
-ETC_20150317_007501	$Siauliai Miners' Village (1)
+ETC_20150317_007501	$Miners' Village (1)
```
Comment (ttgmichael) [inline ETC.tsv]: Should we be deleting Siauliai? The Korean text have it.

File: QUEST_JOBSTEP.tsv [hunk 7]
```diff
@@ -2155,14 +2155,14 @@ QUEST_JOBSTEP_20150317_002154	Meet the Cryomancer Master in Klaipeda
...
-QUEST_JOBSTEP_20150317_002158	Meet the Psychokino Master in Siauliai Miners' Village
-QUEST_JOBSTEP_20150317_002159	Defeat the Werewolf in Siauliai Miners' Village
+QUEST_JOBSTEP_20150317_002158	Meet the Psychokino Master at the Miners' Village
+QUEST_JOBSTEP_20150317_002159	Defeat the Werewolf at the Miners' Village
 QUEST_JOBSTEP_20150317_002160	The Psychokino Master is busy but doesn't want to pass on the chance of defeating the Werewolf to the Linker Master. Defeat the Werewolf in Miners' Village.
 QUEST_JOBSTEP_20150317_002161	Defeated the Werewolf. Go back and report to the Psychokino Master.
 QUEST_JOBSTEP_20150317_002162	Meet the Linker Master
-QUEST_JOBSTEP_20150317_002163	Meet the Linker Master in Siauliai Miners' Village
+QUEST_JOBSTEP_20150317_002163	Meet the Linker Master at the Miners' Village
 QUEST_JOBSTEP_20150317_002164	Get rid of the Werewolf in Siaulai Miner's Village
```
Comment (ttgmichael) [inline QUEST_JOBSTEP.tsv:2164]: Siaulai Miner's Village. you missed one (but again I'm not sure if we want to remove them unless @imcgames wants them removed) ^^

File: QUEST_JOBSTEP.tsv (hunk 3 above)
Comment (terrapin47) [inline QUEST_JOBSTEP.tsv:1631]: "Imbued Evil Energy Pouch"?
I agree, but how should it be worded?

Comment (Sunaries) [inline QUEST_JOBSTEP.tsv:1631]: It's fine in my opinion. It's a pouch imbued with evil energy.

File: QUEST_JOBSTEP.tsv [hunk 8]
```diff
@@ -1085,7 +1085,7 @@ QUEST_JOBSTEP_20150317_001084	Price for the Lost Indulgence [Pardoner]
 QUEST_JOBSTEP_20150317_001085	I'll get you the payment.
 QUEST_JOBSTEP_20150317_001086	{memo X}Checking the wagon/MAKING/3/TRACK_BEFORE/None
 QUEST_JOBSTEP_20150317_001087	Advisor's Choice [Oracle]
-QUEST_JOBSTEP_20150317_001088	Magi purification [Priest]
+QUEST_JOBSTEP_20150317_001088	Magic Purification [Priest]
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: Should be: Evil Energy Purification

File: QUEST_JOBSTEP.tsv (hunk 1 above)
Comment (terrapin47) [inline QUEST_JOBSTEP.tsv]: Guess I'll fix that too =P

File: QUEST_JOBSTEP.tsv (hunk 7 above)
Comment (terrapin47) [inline QUEST_JOBSTEP.tsv:2164]: Hmm, I thought it was a location change, but it's in the wiki too.  I'll leave them in.

**Comment by terrapin47** [comment]
Updated based on feedback.

**Comment by Sunaries** [comment]
Good to merge. :+1:

**Comment by imcgames** [comment]
Thank you :+1:

**Pull request merged**